pyglet/math.py

```diff
@@ -768,7 +768,11 @@ def orthogonal_projection(
         z_near: float,
         z_far: float
     ) -> Mat4T:
-        """Create a Mat4 orthographic projection matrix."""
+        """Create a Mat4 orthographic projection matrix for use with OpenGL.
```

Comment:
I wonder if the detailed explanation is necessary here, as the context and use case should be understood already. 
Perhaps it would be enough to just describe it as a transformation matrix? Something like: 
```Create a 4x4 transformation matrix for orthographic projection.```
or 
```Create a 4x4 orthographic projection transformation matrix.```

What do you think?

Reply:
A projection is a transformation, so I don't think that removes any ambiguity

I don't think it's bad to have an extra sentence to be precise, but I can remove that commit if you would like

Reply:
No worries Tom, I can make some tweaks on my end later, but lets merge this in for now.